pkgs/build-support/cc-wrapper/ld-wrapper.sh

```diff
@@ -63,6 +63,21 @@ fi
 
 extraAfter+=($NIX_@infixSalt@_LDFLAGS_AFTER)
 
+# Choose 32-bit dynamic linker if needed
+if [ -e @out@/nix-support/dynamic-linker-m32 ]; then
+    prev=
+    for p in ${params+"${params[@]}"}; do
```

Comment:
It was determined to be valuable not to iterate over params more times than necessary. Could you merge this loop into the next one? (It has already merged two cases, `[ "$NIX_@infixSalt@_DONT_SET_RPATH" != 1 ]` and `[ "$NIX_@infixSalt@_SET_BUILD_ID" = 1 ]`.)

Reply:
Good idea!

Reply:
Done